examples/third_party_examples/ai_agent_functions_node.md

```diff
@@ -0,0 +1,359 @@
+# Build an AI agent with OpenAI functions and the Node.js SDK
+
+OpenAI functions enable your app to take action based on user inputs. This means that it can, e.g., search the web, send emails, or book tickets on behalf of your users, making it more powerful than a regular chatbot.
+
+In this tutorial, you will build an app that uses OpenAI functions along with the latest version of the Node.js SDK. The app runs in the browser, so you only need a code editor and, e.g., VS Code Live Server to follow along locally. Alternatively, write your code directly in the browser via [this code playground.](https://scrimba.com/scrim/c6r3LkU9)
+
+If you prefer watching screencasts over reading, then you can check out [this scrim, which walks through the code line-by-line:](https://scrimba.com/scrim/co0044b2d9b9b7f5bf16e0391)
+
+<iframe src="https://scrimba.com/scrim/co0044b2d9b9b7f5bf16e0391?embed=openai,no-header" width="100%" height="500"></iframe>
```

Comment:
Not sure if this will render well. We'll see! I plan to add some sort of preview functionality in the future. 

Part of me wants to keep the content consistent across notebooks and link to this rather than embedding it, but I think it's good content and I appreciate the effort you put into it, so I'm curious to see how it looks.

Reply:
Thanks a lot, hoping it'll work!